installation/kie-helm-charts.adoc

```diff
@@ -0,0 +1,409 @@
+
+Follow the command to install the chart with the release name `kie-sandbox`:
+
+[source, shell]
+----
+$ helm install kie-sandbox ./src --values ./src/values-minikube-nginx.yaml
+----
+
+Following message must be displayed on your console.
+
+[source, shell]
+----
+NAME: kie-sandbox
+LAST DEPLOYED: Wed Nov 29 17:09:04 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+NOTES:
+You may need to add the above hostnames to your /etc/hosts file, mapping them to your minikube ip.
+
+Run the following commands:
+  export MINIKUBE_IP=$(minikube ip)
+  echo "\n# Minikube KIE Sandbox Helm Chart hostnames" | sudo tee -a /etc/hosts
+  echo "$MINIKUBE_IP cors-proxy.local" | sudo tee -a /etc/hosts
+  echo "$MINIKUBE_IP extended-services.local" | sudo tee -a /etc/hosts
+  echo "$MINIKUBE_IP kie-sandbox.local" | sudo tee -a /etc/hosts
+----
+
+=== Kubernetes install
+
+Follow the command to install the chart with the release name `kie-sandbox`:
+
+[source, shell]
+----
+$ helm install kie-sandbox ./src --values ./src/values-kubernetes.yaml --set global.kubernetesClusterDomain="<YOUR_KUBERNETES_CLUSTER_DOMAIN>" --set global.kubernetesIngressClass="<YOUR_KUBERNETES_INGRESS_CLASS>"
+----
+
+Following message must be displayed on your console.
+
+[source, shell]
+----
+NAME: kie-sandbox
+LAST DEPLOYED: Wed Nov 29 17:09:04 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+NOTES:
+1. CORS Proxy available at:
+  http://cors-proxy.<YOUR_KUBERNETES_CLUSTER_DOMAIN>
+2. Extended Services available at:
+  http://extended-services.<YOUR_KUBERNETES_CLUSTER_DOMAIN>
+3. KIE Sandbox available at:
+  http://kie-sandbox.<YOUR_KUBERNETES_CLUSTER_DOMAIN>
+----
+
+You need not run any commands. The {CANVAS}
+ can be accessible via https://kie-sandbox.<YOUR_KUBERNETES_CLUSTER_DOMAIN>
+
+=== OpenShift install
+
+First, you may need to get the default OpenShift domain for your routes with the following command:
+
+[source, shell]
+----
+$ oc get ingresses.config cluster --output jsonpath={.spec.domain}
+----
+
+If you do not have access rigths to this configutation, try creating a dummy Route resource and check its domain.
+
+Follow the command to install the chart with the release name `kie-sandbox`:
+
+[source, shell]
+----
+$ helm install kie-sandbox ./src --values ./src/values-openshift.yaml --set global.openshiftRouteDomain="<YOUR_OCP_ROUTE_DOMAIN>"
+----
+
+Following message must be displayed on your console.
+
+[source, shell]
+----
+NAME: kie-sandbox
+LAST DEPLOYED: Wed Nov 29 17:09:04 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+NOTES:
+1. CORS Proxy available at:
+  https://cors-proxy.<YOUR_OCP_ROUTE_DOMAIN>
+2. Extended Services available at:
+  https://extended-services.<YOUR_OCP_ROUTE_DOMAIN>
+3. KIE Sandbox available at:
+  https://kie-sandbox.<YOUR_OCP_ROUTE_DOMAIN>
+----
+
+You need not run any commands. The BAMOE CANVAS must be accessible via https://kie-sandbox.<YOUR_OCP_ROUTE_DOMAIN>
+
+== Installing a released version from the OCI registry:
```

Comment:
I think we should move this up, as installing from source should be an alternative to installing directly from the released version from the OCI registry